### PR TITLE
feat: add brazilian portuguese language

### PR DESCRIPTION
**Pull request by albertosgz**
Ticket: https://oat-sa.atlassian.net/browse/AUT-2995

## What has been done

Added brazilian portuguese and append `(Portugal)` to current portuguese.

image

### Requirements 

**Required** https://github.com/oat-sa/tao-core/pull/3791 to get new language option on select.

### CODE REVIEW

**Review by bziondik — APPROVED** [review]
No need to update composer dependencies - usually it is some features that implemented across extensions
No need to put bundles, they will be created on autorelease

- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on plauground (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Comment by bziondik** [comment]
CI fails maybe because of composer changes

**Comment by mike-ionut-mihai-sandu-tao** [comment]
let's merge this PR please

File: composer.json [hunk 1]
```diff
@@ -19,17 +19,17 @@
...
-        "oat-sa/tao-core": ">=51.2.0",
-        "oat-sa/extension-tao-delivery": ">=15.0.0",
-        "oat-sa/extension-tao-lti": ">=15.4.1",
-        "oat-sa/extension-tao-delivery-rdf": ">=14.0.0",
-        "oat-sa/extension-tao-proctoring": ">=20.0.0",
-        "oat-sa/extension-tao-testqti": ">=41.0.0",
-        "oat-sa/extension-tao-outcome": ">=13.0.0",
-        "oat-sa/extension-tao-testqti-previewer": ">=3.0.0",
-        "oat-sa/extension-tao-outcomeui": ">=11.0.0"
+        "oat-sa/tao-core": "dev-feature/AUT-2995",
```
Comment (mike-ionut-mihai-sandu-tao) [inline composer.json:24]: please revert

**Review by mike-ionut-mihai-sandu-tao — CHANGES_REQUESTED** [review]

**Pull request merged**